unit/libcprover-cpp/verification_resultt.cpp

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) 2023 Fotis Koutoulakis for Diffblue Ltd.
+
+/// \file Integration tests for the the interface of the class for verification results
+///       produced by various verification engines (classes downstream of
+///       goto_verifiert).
+
+#include <libcprover-cpp/verification_result.h>
+
+#include <iostream>
+#include <sstream>
+#include <string>
+
+#include "../catch/catch.hpp"
+#include "../testing-utils/get_goto_model_from_c.h"
```

Comment:
:no_entry_sign: This is introducing a transitory include of `goto-programs/goto_model.h`  which is not part of the API. It must be possible to test the API without using the rest of the codebase.